src/sentry/static/sentry/app/views/projectDetail/projectCharts.tsx

```diff
@@ -58,32 +79,76 @@ class ProjectCharts extends React.Component<Props, State> {
   get displayModes(): SelectValue<string>[] {
     const {organization} = this.props;
     const hasPerformance = organization.features.includes('performance-view');
+    const hasDiscover = organization.features.includes('discover-basic');
+    const noPerformanceTooltip = t(
+      'This view is only available with Performance Monitoring.'
+    );
+    const noDiscoverTooltip = t('This view is only available with Discover feature.');
+
     return [
       {
         value: DisplayModes.APDEX,
         label: t('Apdex'),
-        disabled: !hasPerformance,
+        disabled:
+          this.otherActiveDisplayModes.includes(DisplayModes.APDEX) || !hasPerformance,
         tooltip: hasPerformance
           ? getTermHelp(organization, 'apdex')
-          : t('This view is only available with Performance Monitoring.'),
+          : noPerformanceTooltip,
+      },
+      {
+        value: DisplayModes.FAILURE_RATE,
+        label: t('Failure Rate'),
+        disabled:
+          this.otherActiveDisplayModes.includes(DisplayModes.FAILURE_RATE) ||
+          !hasPerformance,
+        tooltip: hasPerformance
+          ? getTermHelp(organization, 'failureRate')
+          : noPerformanceTooltip,
+      },
+      {
+        value: DisplayModes.TPM,
+        label: t('Transactions Per Minute'),
+        disabled:
+          this.otherActiveDisplayModes.includes(DisplayModes.TPM) || !hasPerformance,
+        tooltip: hasPerformance ? getTermHelp(organization, 'tpm') : noPerformanceTooltip,
+      },
+      {
+        value: DisplayModes.ERRORS,
+        label: t('Daily Errors'),
+        disabled:
+          this.otherActiveDisplayModes.includes(DisplayModes.ERRORS) || !hasDiscover,
+        tooltip: hasDiscover ? undefined : noDiscoverTooltip,
+      },
+      {
+        value: DisplayModes.TRANSACTIONS,
+        label: t('Daily Transactions'),
+        disabled:
+          this.otherActiveDisplayModes.includes(DisplayModes.TRANSACTIONS) ||
+          !hasPerformance,
+        tooltip: hasPerformance ? undefined : noPerformanceTooltip,
       },
     ];
   }
 
   get summaryHeading() {
     switch (this.displayMode) {
+      case DisplayModes.ERRORS:
+        return t('Total Errors');
       case DisplayModes.APDEX:
+      case DisplayModes.FAILURE_RATE:
+      case DisplayModes.TPM:
+      case DisplayModes.TRANSACTIONS:
       default:
         return t('Total Transactions');
     }
   }
 
   handleDisplayModeChange = (value: string) => {
```

Comment:
I think I'll add analytics here so that we know what charts are people most interested in.

Reply:
done